R/msnset2se.R

```diff
@@ -0,0 +1,48 @@
+#' SummarizedExperiment to MSnSet object conversion
+#'
+#' \code{se2msnset} generates an MSnSet object from an SummarizedExperiment object.
+#'
+#' @param se SummarizedExperiment,
+#' Object which will be turned into an MSnSet object.
+#' @return MSnSet object.
+#' @export
+se2msnset <- function(se) {
+  # Check input
+  assertthat::assert_that(inherits(se, "SummarizedExperiment"))
+
+  # Extract expression, feature and pheno data
+  raw <- SummarizedExperiment::assay(se)
+  feat_data <- data.frame(SummarizedExperiment::rowData(se))
+  rownames(feat_data) <- se@NAMES
```

Comment:
You should not access slots directly. Why not use `names(se)` here? I am not very familiar with `SummarizedExperiment` yet but are you sure that `rownames(rowData(se))` is never set?